CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+Welcome to Kùzu! We are excited that you are interested in contributing to Kùzu.
+Before submitting your contribution though, please make sure to take a moment and read through the following guidelines.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by a [Code of Conduct](CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. 
+Please report unacceptable behavior to [contact@kuzudb.com](mailto:contact@kuzudb.com).
+
+## Outside Contributors
+Join our project's [Slack workspace](https://join.slack.com/t/kuzudb/shared_invite/zt-1n67h736q-E3AFGSI4w~ljlFMYr3_Sjg) for real-time communication with the core team and other contributors.
+If you have a question or need help, feel free to ask in the appropriate channel or create an issue.
+And be aware that by contributing to this project, you agree to release your contributions under the [MIT License](LICENSE).
```

Comment:
I would replace this with this blurb:

```
When you contribute code, you affirm that the contribution is your original work and that you license the work to the project under the project's open source license. Whether or not you state this explicitly, by submitting any copyrighted material via pull request, email, or other means you agree to license the material under the project's open source license and warrant that you have the legal authority to do so.
```
I copied this from Spark's github and had seen it in several places.